fundingmanager.go

```diff
@@ -2429,6 +2435,11 @@ func (f *fundingManager) handleFundingLocked(fmsg *fundingLockedMsg) {
 		return
 	}
 
+	// As this is the earliest possible time that the channel can be used
```

Comment:
I'm just wondering.. should we really be adding the link to the switch if we haven't yet received `fundinglocked`?

Reply:
We need to in order to be able to process the chan reest message sent by the remote peer upon reconnection. We had cross-impl issues with this in the past until we moved to always loading the channel in the link pre funding locked.

Reply:
Riight...